manifest.json

```diff
@@ -716,6 +716,24 @@
       "useCaseName": "Multi-step workflow"
     }
   ],
+  "rust": [
+    {
+      "directory": "rust/cookiecutter-aws-sam-rust-hello",
+      "displayName": "Hello World Example",
+      "dependencyManager": "cargo",
+      "appTemplate": "hello-world",
+      "packageType": "Zip",
+      "useCaseName": "Serverless API"
```

Comment:
This would be changed to `Hello World` use case with this PR: https://github.com/aws/aws-sam-cli-app-templates/pull/146